### PR TITLE
Set proper exit status

### PR DESCRIPTION
**Pull request by loucash**
I am using `riak_test` on Travis CI and properly set exit status would be very useful.

### CODE REVIEW

**Comment by mjbrender** [comment]
This is fantastic @loucash! Thank you for it. Let me check w/the team before making any merges - we're a little behind on PRs for `riak_test` and appreciate your patience.

**Comment by loucash** [comment]
Cool, thanks :+1:

File: src/riak_test_escript.erl [hunk 1]
```diff
@@ -373,6 +373,15 @@ print_summary(TestResults, CoverResult, Verbose) ->
     end,
     ok.
 
+proper_exit_status(TestResults) ->
+    FailCount = length(lists:filter(fun(X) -> proplists:get_value(status, X) =:= fail end, TestResults)),
```
Comment (seancribbs) [inline src/riak_test_escript.erl]: A list comprehension would be clearer here, and you also don't need to compute the length in the following case expression if you just match the empty list.

**Comment by seancribbs** [comment]
:+1:

**Pull request merged**